app/models/lunch.rb

```diff
@@ -3,12 +3,23 @@ class << self
     include Feedjira
     include Nokogiri
     include OpenURI
+    include JSON
+
+=begin
+    Allergens:
+      0 no allergens
+      1 gluten
+      2 lactose
+      3 egg
+      4 rennet
+      6 fish
+=end
```

Comment:
These could be constants instead. And then the constants could be mapped to the allergen images below.